beacon_node/src/config.rs

```diff
@@ -148,6 +148,10 @@ pub fn get_config<E: EthSpec>(
         client_config.http_api.allow_sync_stalled = true;
     }
 
+    if let Ok(Some(cache_size)) = clap_utils::parse_optional(cli_args, "shuffling-cache-size") {
```

Comment:
We should error if the value is invalid, currently this will proceed silently. My bad with the previous suggestion.

```suggestion
    if let Some(cache_size) = clap_utils::parse_optional(cli_args, "shuffling-cache-size")? {
```